tasks.py

```diff
@@ -221,6 +221,12 @@ def makemigrations(ctx):
     manage(ctx, "makemigrations")
 
 
+@task(aliases=["docker-makemigrations-dryrun"])
```

Comment:
Not sure if there's a better way to achieve this, but I use the `--dry-run` regularly

Reply:
I don't think I ever do, but if this makes someone's life easier: \o/